src/lib/datepicker/calendar.ts

```diff
@@ -340,6 +340,9 @@ export class MatCalendar<D> implements AfterContentInit, AfterViewChecked, OnDes
             (this.currentView == 'year' ? this.yearView : this.multiYearView);
 
     view.ngAfterContentInit();
+
+    // to be removed once test works
+    console.log(view._dateAdapter.today());
```

Comment:
@mmalerba I am still getting today's date instead of the fake one. Maybe I missed something.

Reply:
test output removed in 
https://github.com/angular/material2/pull/11821/commits/cb84cbe9c2fc5a02da3ba045ec1d91d4afef55e8